static/app/views/performance/landing/widgets/components/widgetHeader.tsx

```diff
@@ -25,6 +30,7 @@ export function WidgetHeader<T extends WidgetDataConstraint>(
           ) : (
             <TextOverflow>{title}</TextOverflow>
           )}
+          {isStarfishDBWidget && <FeatureBadge type="alpha" />}
```

Comment:
Could we adjust the alignment of the badge?

<img width="885" alt="image" src="https://github.com/getsentry/sentry/assets/18689448/41b834f4-c442-43c6-a821-ef7e604b2679">

Reply:
Done.